src/observable/ErrorObservable.ts

```diff
@@ -63,7 +63,7 @@ export class ErrorObservable<T> extends Observable<any> {
     subscriber.error(error);
   }
 
-  constructor(public error: T, private scheduler?: IScheduler) {
+  constructor(public readonly error: any, private readonly scheduler?: IScheduler) {
```

Comment:
Is this not going to be a breaking change for some users? I feel like we've done this in the past, but I can't remember the verdict. @staltz? What are your thoughts?

For now, @kwonoj, can we exclude this change from this PR?